### ISSUE TITLE
How to add cars to the CARLA dataset?

### ISSUE THREAD

**Issue by wangkke** [ethliup/UnrealCV#2]
Now I can use the dataset to render image for any scene and building, but next, i want to know what should I do to add cars to the CARLA dataset ?

**Comment by ethliup** [comment]
Actually I am not quite sure how to do this. I have not tried it. But I think it should be possible. I only use it to render images from static scenes

**Comment by wangkke** [comment]
I want to ask,  when you open the town dataset with the UE4, can you see statistic cars on the road?
 Or is there any operation to make cars on the road?
 Because in your paper,  I can see some pictures which contain many cars.
And i want to render some images which contain cars too.
Thanks for your reply!

**Comment by ethliup** [comment]
Yes, there will have several static cars. Especially inside town-05. It is already there.

**Comment by wangkke** [comment]
Hello, thank you very much for the very comprehensive dataset. But the carla dataset  is relatively large and requires 16G,and i encountered some network problems during the download process and have not downloaded the dataset yet. So I would like to ask if you can provide a separate dataset containing only the Town-05 scene. Thanks!

**Comment by ethliup** [comment]
ok, I will try to divide it into several small pieces.

**Comment by wangkke** [comment]
Thank you very much!
